### ISSUE TITLE
warn users that `lfs: true` can bloat bandwidth consumption

### ISSUE THREAD

**Issue by maxheld83** [actions/checkout#834]
We learned recently that setting `lfs: true` can really bloat the bandwidth consumption of git lfs data packs, because this setting will make the checkout action download *all* pointers on *every* commit (apparently?).

It would be great if either:

1. this wouldn't count towards the quota, since this is presumably, all inside gh datacenter
1. there would be a transparent caching layer behind it to avoid this (as per #165)
1. barring 1 and 2, the docs should warn users that this will happen.

**Comment by klausbadelt** [comment]
Agree with all of the above. As temporary workaround we're using [mshick/add-pr-comment](https://github.com/marketplace/actions/add-pr-comment) currently.

**Comment by kelle** [comment]
Yes, user should be warned but even better would be adding a `cache: True` option to the next version of checkout.

In the meantime, the implementation provided by @nschloe worked for me: https://github.com/actions/checkout/issues/165#issuecomment-1639209867
